src/Kernel32.Desktop/Kernel32+SafeLibraryHandle.cs

```diff
@@ -16,6 +16,8 @@ public static partial class Kernel32
         /// </summary>
         public class SafeLibraryHandle : SafeHandle
         {
+            public static readonly SafeLibraryHandle Null = new SafeLibraryHandle(new IntPtr(0), false);
```

Comment:
You don't have to fix this, but as a nit: `IntPtr.Zero` is what we typically use instead of `new IntPtr(0)`